lib/worker.js

```diff
@@ -164,7 +164,7 @@ Worker.prototype._start = function() {
         self.serviceReturns = res;
         // Make sure that only JSON-serializable values are returned.
         try {
-            return [JSON.parse(JSON.stringify(res))];
+            return JSON.parse(JSON.stringify(res));
```

Comment:
Yes, but for consistency with the master it's expected to be an array of arrays. In the case of running without workers, this will be the exact return value.